tests/src/test/java/de/sovity/edc/e2e/utils/MockedAsset.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.e2e.utils;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public record MockedAsset(
+    String assetId,
+    AtomicInteger accesses
```

Comment:
docs, what does this all mean, what is it for?

the more obstruse, short and "generally sounding" something has been left, the higher is the push to have to document what it is and what it was made for

Reply:
Should be clearer by looking at how it's used in tests, but doc added.